aten/src/ATen/native/native_functions.yaml

```diff
@@ -7787,6 +7787,23 @@
   use_c10_dispatcher: full
   variants: function
 
+# Would like this to be "int[1]? dim=[-2,-1]" but this fails with an error at import time
+- func: fft_fft2(Tensor self, int[1]? s=None, int[1] dim=[-2,-1], str? norm=None) -> Tensor
```

Comment:
This is the only reason I've marked this as draft. Technically, you can use `np.fft.fft2(x, axes=None)` as a weird alternative to `np.fft.fftn(x)`. Supporting this would require more changes to the code generation to support optional arrays with defaults other than `None`. 

This isn't really the intended way to use these functions though, so it might be okay to break compatibility here.

Reply:
As per our discussion, this is fine since we're raising an error instead of giving the wrong answer compared to NumPy.